python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -1839,6 +1840,46 @@ def __getitem__(self, key):
             group_keys=self._group_keys,
         )
 
+    def _reduce(
```

Comment:
I did overload methods because i want [This page](https://docs.rapids.ai/api/cudf/nightly/user_guide/pandascompat)
to become more readable.
like this
![image](https://user-images.githubusercontent.com/58826535/215684295-b9511fe8-b2d4-4ac7-baaf-ff528f1cb0f9.png)

instead of using same ClassName same methods.

Reply:
I am -1 on this change since it violates OAOO for an only very marginal gain.

Reading the `Reducible` mixin, I think that `{cls}` will be formatted to the class name of the implementing class. So can we drop these additions, and modify the groupby change (see above).

Reply:
Thank you for this, it took me so long to find how to change these I will experiment and commit change soon.